### PR TITLE
style: secondary button

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

I noticed that the "color" of the secondary button was not the contrast one in the Figma file defined by the designer.

This PR tweaks the secondary button to be closer to what they have in mind.

### CODE REVIEW

File: src/lib/styles/mixins/_overlay.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@mixin colors {
```
Comment (lmuntaner) [inline src/lib/styles/mixins/_overlay.scss:1]: why the name `overlay` for this mixin?

**Review by lmuntaner — APPROVED** [review]
LGTM, just a question

File: src/lib/styles/mixins/_overlay.scss (hunk 1 above)
Comment (peterpeterparker) [reply src/lib/styles/mixins/_overlay.scss:1]: because it defines the background colors of the toast and modal

Comment (lmuntaner) [reply src/lib/styles/mixins/_overlay.scss:1]: aahh, ok, and they are overlays. Understood.

**Pull request merged**